migration/config-repair/remove_incorrect_configs.py

```diff
@@ -0,0 +1,20 @@
+import tables
+from shutil import copyfile
+
+
+def remove_configs(filename, cluster, stationnumber):
+    """
+    Remove all but the first configuration row
+    """
+    
+    copyfile(filename, filename+'_backup')
+
+    with tables.open_file(filename, "a") as data:
+        config = data.get_node("/hisparc/cluster_"+cluster+"/station_"+str(stationnumber)+"/config")
```

Comment:
`data.config('/hisaprc/cluster_%s/station_%d/config' % (cluster, stationnumber))` is much more readable. IMHO